README.md

```diff
@@ -1,2 +1,34 @@
 # openshift-launchpad
 A project template to accelerate onboarding to the OpenShift platform
+
+## Table of Contents
+
+1. [Getting Help or Reporting an Issue](#getting-help-or-reporting-an-issue)
+1. [How to Contribute](#how-to-contribute)
+1. [License](#license)
```

Comment:
1,2,3 ?

Reply:
Cool markdown feature - these get interpreted to the right numbers when viewed :)

Reply:
How intuitive 🤣

Reply:
No kidding lol